src/types.ts

```diff
@@ -58,3 +70,12 @@ export type ItemValidationReviewStatus = {
   id: string;
   name: string;
 };
+
+export type ImageClassifierResponse = {
```

Comment:
This is tied to the classifier you used, maybe give it a name following the classifier name.